ios/ReactNativeCameraKit/CameraView.swift

```diff
@@ -33,6 +33,7 @@ class CameraView: UIView {
     // props
     // camera settings
     @objc var cameraType: CameraType = .back
+    @objc var resizeMode: ResizeMode = .cover
```

Comment:
As per DavidBertet's suggestion, going to use contain by default.
```suggestion
    @objc var resizeMode: ResizeMode = .contain
```